index.html

```diff
@@ -8,24 +8,26 @@
     <title>My Portfolio</title>
 </head>
 <body>
-    <header class="text-center">
-        <nav class="navbar navbar-expand-lg navbar-light bg-light" id="navbar-example2">
-            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo01" aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
-                <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
-                    <li class="nav-item"><a class="nav-link" href="#scrollspyHeading1">Portfolio</a></li>
-                    <li class="nav-item"><a class="nav-link" href="#scrollspyHeading2">About</a></li>
-                    <li class="nav-item"><a class="nav-link" href="#scrollspyHeading3">Contact</a></li>
-                </ul>
+    <header class="text-center my-5">
+        <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark" id="navbar-example2">
+            <div class="container-fluid">
+                <button class="navbar-toggler justify-content-end" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo01" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+                    <ul class="navbar-nav nav nav-pills me-auto mb-2 mb-md-0">
+                        <li class="nav-item"><a class="nav-link" href="#scrollspyHeading1">Portfolio</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#scrollspyHeading2">About</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#scrollspyHeading3">Contact</a></li>
+                    </ul>
+                </div>
```

Comment:
### Consider incorporating semantic tags into your HTML markup like the ```section``` tage.
- Semantic tags provide meaningful structure and context to the content of a web page, allowing search engines and assistive technologies to better understand the purpose and hierarchy of various elements. By utilizing semantic tags, you can improve both the user experience and search engine optimization (SEO) of your website.